pyk/src/pyk/tests/test_utils.py

```diff
@@ -0,0 +1,18 @@
+from unittest import TestCase
+
+from ..utils import deconstruct_short_hash
+
+
+class KCFGTestCase(TestCase):
```

Comment:
```suggestion
class ShortHashTest(TestCase):
```

Reply:
Please add a few tests for successful deconstruction (prefix with / without the dots, suffix, both prefix and suffix, full hash, etc).